scheduler/pkg/kafka/config/tls.go

```diff
@@ -34,8 +34,10 @@ const (
 func AddKafkaSSLOptions(config kafka.ConfigMap) error {
 	protocol := tls.GetSecurityProtocolFromEnv(tls.EnvSecurityPrefixKafka)
 	switch protocol {
-	case tls.SecurityProtocolSSL:
+	case tls.SecurityProtocolTLS:
 		return setupTLSAuthentication(config)
+	case tls.SecurityProtocolSSL:
```

Comment:
It may be a bit confusing to have both `TLS` and `SSL` as different protocol options. Given that, `SSL` seems to be just doing mTLS - should we rename the `SSL` flag to `MTLS`?

Reply:
Upon further investigation, I would propose to use a single SSL option.
Reason: if I read the code correct, the security parameter is directly mapped to the [Kafka Security Protocol ](https://kafka.apache.org/25/javadoc/org/apache/kafka/common/security/auth/SecurityProtocol.html#SSL), at least in the dataflow component.
Kafka does not distinguish between TLS encryption (1-way) and authentication (2-way/mutual/client), as the protocol in both cases is TLS.

Reply:
+1

Reply:
Yes, I would leave it only as single `security.kafka.protocol: SSL` that corresponds to `security.protocol=SSL` option for the Kafka client. Then everything should be driven by:
- `security.kafka.ssl.client.secret`
- `security.kafka.ssl.client.brokerValidationSecret`